hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/index/functional/BaseHoodieIndexClient.java

```diff
@@ -61,6 +71,8 @@ public void register(HoodieTableMetaClient metaClient, HoodieIndexDefinition ind
    */
   public abstract void create(HoodieTableMetaClient metaClient, String indexName, String indexType, Map<String, Map<String, String>> columns, Map<String, String> options) throws Exception;
 
+  public abstract void createOrUpdateColStatsIndex(HoodieTableMetaClient metaClient, List<String> columnsToIndex);
```

Comment:
how about rename to `createOrUpdateColumnStatsIndexDefinition`? The method is not actually creating/updating the stats right.

Reply:
👍